### PR TITLE
ROX-18481: Add CENTRAL_EVENTS feature flag

### PR DESCRIPTION
**Pull request by dhaus67**
## Description

As a preparation for the upcoming Central events feature, this PR introduces the feature flag for it for both backend as well as frontend parts.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

n/a

### CODE REVIEW

File: pkg/features/list.go [hunk 1]
```diff
@@ -44,4 +44,7 @@ var (
 
 	// ComplianceEnhancements enables APIs and UI pages for Compliance 2.0
 	ComplianceEnhancements = registerFeature("Compliance enhancements", "ROX_COMPLIANCE_ENHANCEMENTS", false)
+
+	// ExposeCentralEvents enables APIs and UI pages for exposing Central events.
+	ExposeCentralEvents = registerFeature("Expose Central events", "ROX_CENTRAL_EVENTS", false)
```
Comment (stehessel) [inline pkg/features/list.go:49]: So the feature flag only gates the API being exposed, but in the background the events are still being collected and persisted? Shouldn't we have a feature flag for the entire central events backend?

Comment (dhaus67) [reply pkg/features/list.go:49]: Looking at the other examples, this is what we generally write as generic comment when adding feature flags.

When it's disable, we won't do anything on the backend (nothing will be persisted nor intercepted), I can change the comment if you want.

Comment (stehessel) [reply pkg/features/list.go:49]: Maybe `s/Expose/Enable`?

**Review by stehessel — APPROVED** [review]

File: ui/apps/platform/src/types/featureFlag.ts [hunk 2]
```diff
@@ -6,4 +6,5 @@ export type FeatureFlagEnvVar =
     | 'ROX_COMPLIANCE_ENHANCEMENTS'
     | 'ROX_VULN_MGMT_REPORTING_ENHANCEMENTS'
     | 'ROX_VULN_MGMT_WORKLOAD_CVES'
+    | 'ROX_CENTRAL_EVENTS'
```
Comment (pedrottimark) [inline ui/apps/platform/src/types/featureFlag.ts:9]: What’s done it done, but in the future please add in alphabetical order (per comment).

It somewhat reduces merge conflicts if multiple people add or delete around the same time.

Comment (dhaus67) [reply ui/apps/platform/src/types/featureFlag.ts:9]: Ah, that's embarrasing, here's the fix: https://github.com/stackrox/stackrox/pull/7321
And thanks for catching it!

**Pull request merged**